presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -167,6 +167,7 @@ Property Name                                      Description
 
 ``security.config-file``                           Path of config file to use when ``hive.security=file``.
                                                    See :ref:`hive-file-based-authorization` for details.
+``hive.non-managed-table-writes-enabled``          Enable writes to non-managed Hive tables.                    ``false``
```

Comment:
What do you think about dropping the `-enabled` from the property name? This will be consistent with `bucket-execution` and `bucket-writing`. Or do you think the suffix adds value?